modules/dxp/apps/analytics/analytics-reports-web/src/main/java/com/liferay/analytics/reports/web/internal/constants/AnalyticsReportsPortletKeys.java

```diff
@@ -23,4 +23,7 @@ public class AnalyticsReportsPortletKeys {
 		"com_liferay_analytics_reports_web_internal_portlet_" +
 			"AnalyticsReportsPortlet";
 
+	public static final String SESSION_CLICKS_KEY =
```

Comment:
Hey @RenanUmV, as this constant is not a portletKey we should put it in another place, like `AnalyticsReportsConstants` class.

Thx